notification/src/main/java/com/iluwatar/notification/Error.java

```diff
@@ -0,0 +1,23 @@
+package com.iluwatar.notification;
+
+public class Error {
```

Comment:
It's good practice to provide Javadoc for each class

Reply:
`Error` as a name can be a little bit too generic. Please consider alternatives.